Content.Client/GameTicking/Managers/ClientGameTicker.cs

```diff
@@ -70,7 +74,16 @@ private void LateJoinStatus(TickerLateJoinStatusEvent message)
         private void UpdateJobsAvailable(TickerJobsAvailableEvent message)
         {
             _jobsAvailable.Clear();
-            _jobsAvailable.AddRange(message.JobsAvailable);
+            _stationNames.Clear();
+            foreach (var (key, value) in message.JobsAvailableByStation)
+            {
+                _jobsAvailable.Add(key, value);
+            }
+
+            foreach (var (key, value) in message.StationNames)
+            {
+                _stationNames.Add(key, value);
+            }
```

Comment:
Given that the client never needs to modify these, why not just make these public `IReadOnlyDictionary`s and just directly use
 ```cs
JobsAvailable = message.JobsAvailableByStation;
StationNames = message.StationNames;
```